src/msp_displayport.c

```diff
@@ -1629,115 +1633,99 @@ void set_vtx_param()
 }
 
 #ifdef INIT_VTX_TABLE
-void InitVtxTable()
-{
-    uint8_t i,j;
+static const uint8_t bf_vtx_band_table[6][31] = {
```

Comment:
This same type of conversion should be applied to `DM6300_init7` looks like it's inlined i2c commands

Reply:
i dont quite follow, could you elaborate on that?
like split the regmap and compose in-function?
or set a ptr and run the `DM6300_write_reg_map` function on the very end?

Reply:
I see why you are confused, please ignore my comment, I failed to actually load the diff for dm6300.c and will now think about my life choices lol